### PR TITLE
fix: fix eslint issues

### PR DESCRIPTION
**Pull request by Hemanthghs**

### CODE REVIEW

File: frontend/src/utils/util.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 export const convertPaginationToParams = (pagination) => {
-  let result = "";
+  let result = '';
   if (
```
Comment (charymalloju) [inline frontend/src/utils/util.js:3]: I think, you could rewrite this logic `if (!pagination || (!pagination.key && !pagination.limit)) {
  return '';
}`

File: frontend/src/utils/util.js [hunk 2]
```diff
@@ -22,13 +22,13 @@ export const convertPaginationToParams = (pagination) => {
 };
 
 export const convertPaginationToParamsOffset = (pagination) => {
-  let result = "";
+  let result = '';
   if (
     pagination === undefined ||
```
Comment (charymalloju) [inline frontend/src/utils/util.js:27]: follow the above comment

File: frontend/src/components/popups/WalletPage.tsx [hunk 3]
```diff
@@ -31,7 +31,7 @@ const Walletpage = ({
             <div
```
Comment (charymalloju) [inline frontend/src/components/popups/WalletPage.tsx:31]: I suggest defining a constant for the networks array and mapping it. This way, if a new wallet is added in the future, there's no need to create another div.

File: frontend/src/components/PermanentSidebar.tsx [hunk 4]
```diff
@@ -1,59 +1,58 @@
...
+    link: '/groups',
   },
-];
+]
 
 const PermanentSidebar = ({ children }: { children: React.ReactNode }) => {
-  const pathname = usePathname();
-  const pathParts = pathname.split("/");
-  const selectedPart = getSelectedPartFromURL(pathParts);
+  const pathname = usePathname()
```
Comment (charymalloju) [inline frontend/src/components/PermanentSidebar.tsx:53]: semicolons are missed

File: frontend/src/components/PermanentSidebar.tsx [hunk 5]
```diff
@@ -1,59 +1,58 @@
...
+  const pathParts = pathname.split('/')
```
Comment (charymalloju) [inline frontend/src/components/PermanentSidebar.tsx:54]: here

File: frontend/src/components/PermanentSidebar.tsx [hunk 6]
```diff
@@ -1,59 +1,58 @@
...
+  const selectedPart = getSelectedPartFromURL(pathParts)
```
Comment (charymalloju) [inline frontend/src/components/PermanentSidebar.tsx:55]: here

File: frontend/src/components/PermanentSidebar.tsx [hunk 7]
```diff
@@ -1,59 +1,58 @@
...
   {
-    name: "Groups",
-    icon: "/groups-icon.svg",
-    link: "/groups",
+    name: 'Groups',
+    icon: '/groups-icon.svg',
+    link: '/groups',
   },
-];
+]
```
Comment (charymalloju) [inline frontend/src/components/PermanentSidebar.tsx:50]: semicolon

File: frontend/src/components/PermanentSidebar.tsx [hunk 8]
```diff
@@ -1,59 +1,58 @@
-"use client";
+'use client'
 
-import Image from "next/image";
-import Link from "next/link";
-import { usePathname } from "next/navigation";
-import React from "react";
-import TopNav from "./TopNav";
-import { getSelectedPartFromURL } from "../utils/util";
+import Image from 'next/image'
```
Comment (charymalloju) [inline frontend/src/components/PermanentSidebar.tsx:3]: It appears that the entire imports in the code do not contain any semicolons.

**Review by charymalloju — APPROVED** [review]

**Review by aleem1314 — APPROVED** [review]

**Pull request merged**